src/main/webapp/result.html

```diff
@@ -0,0 +1,211 @@
+<!-- Copyright 2020 Google LLC
```

Comment:
Let's send this file with JS files.
Similarly other UI related files.

Only keep server related files here.